docs/book/src/faq/faq.md

```diff
@@ -0,0 +1,43 @@
+# FAQ
+
+Frequently Ask Question
+
+## How to use Kubernetes client in webhook?
+
+You can declare the Kubernetes client in webhook from the controller manager easily.
+
+<details>
+  <summary>Read more</summary>
+
+To begin, you need to import the client and declare the variable:
+
+```go
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	C client.Client
+)
+```
+
+Next, it is very simple to fill the previous client variable from the controller manager.
+
+```go
+func (r *MyAwesomeCRD) SetupWebhookWithManager(mgr ctrl.Manager) error {
```

Comment:
@JulienBreux this is scaffolded by kubebuilder by default, with the `create webhook` command. I don't think this needs to be done manually. 

According to the docs, 
```
`kubebuilder create webhook --group batch --version v1 --kind CronJob --defaulting --programmatic-validation`
This will scaffold the webhook functions and register your webhook with the manager in your main.go for you.
```

Reply:
Thanks, yes, I known.

This FAQ entry was just here to demonstrate how to use client in this generated webhook file.
Or "after" the generation.